tests/utilities/test_reader.py

```diff
@@ -233,6 +233,26 @@ def test_ensightreader_timepoints():
     assert reader.active_time_value == 1.0
 
 
+def test_dcmreader():
+    # Test reading folder (image stack)
+    folder = examples.download_dicom_stack(load=False)
+    reader = pyvista.DICOMReader(folder)  # ``get_reader`` doesn't support folders
+    assert isinstance(reader, pyvista.DICOMReader)
+    assert reader.filename == folder  # TODO: Add folder support to ``BaseReader``?
+
+    mesh = reader.read()
+    assert isinstance(mesh, pyvista.UniformGrid)
+    assert all([mesh.n_points, mesh.n_cells])
+
+    # Test reading single file
+    import os
+
+    filename = os.path.join(folder, "1-1.dcm")
+    reader = pyvista.get_reader(filename)
+    assert isinstance(reader, pyvista.DICOMReader)
+    assert reader.filename == filename
+
```

Comment:
We should also test that the mesh after reading it in is valid.

Reply:
I will add

```python
    mesh = reader.read()
    assert isinstance(mesh, pyvista.UniformGrid)
    assert all([mesh.n_points, mesh.n_cells])
```